Pilot/DirectoryService.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  DirectoryService.swift
+//  Pilot
+//
+//  Created by Nick Eckert on 6/28/16.
+//  Copyright © 2016 Sanction. All rights reserved.
+//
+
+import Cocoa
+import FileKit
+
+class DirectoryService: NSObject {
+
+  static func getFilesFromPath(platformType: PlatformType, caller: FileService) -> [LocalFile]? {
```

Comment:
When you handle the return value from this function, are you able to do something else if it is `nil`? I was wondering which would be better, returning `nil` or an empty list. `nil` is probably better if we know something went wrong, just wanted to comment and see your thoughts.